.github/workflows/wheels.yml

```diff
@@ -0,0 +1,53 @@
+name: RMM wheels
+
+on:
+  push:
```

Comment:
Having this action run on `push: pull-request` was only to run this PR itself while we iterated on it.

When this workflow is merged to the default branch, it's only expected to be called by `workflow_call`, from the nightly wheel building pipeline in the external actions repo: https://github.com/rapidsai/actions/blob/main/.github/workflows/pip-wheels-pipeline.yml#L33

Reply:
Good point, I'll remove.